File: ios/Classes/view/banner_yandex_ad/BannerYandexAdViewArguments.swift
//
//  BannerViewArguments.swift
//  flutter_yandex_ads_plus
//
//  Created by apple on 21.10.2022.
//

import Foundation

class BannerYandexAdViewArguments {
    let uid: String
    let adId: String
    let height: Int
    let width: Int
    let parameters: AdParameters?
    let settings: BannerYandexAdViewSettings

    init(
        uid: String,
        adId: String,
        height: Int,
        width: Int,
        parameters: AdParameters?,
        settings: BannerYandexAdViewSettings
    ) {
        self.uid = uid
        self.adId = adId
        self.height = height
        self.width = width
        self.parameters = parameters
        self.settings = settings
    }
}

File: android/src/main/kotlin/ru/kardenvan/flutter_yandex_ads_plus/ads/banner_yandex_ad/BannerYandexAdViewArguments.kt
package ru.kardenvan.flutter_yandex_ads_plus.ads.banner_yandex_ad

import android.util.Size
import ru.kardenvan.flutter_yandex_ads_plus.ads.AdParameters

data class BannerYandexAdViewArguments(
    val uid: String,
    val adId: String,
    val size: Size,
    val parameters: AdParameters?
)